contracts/JBController.sol

```diff
@@ -443,30 +431,31 @@ contract JBController3_1 is
         requirePermissionAllowingOverride(
             _holder,
             _projectId,
-            JBOperations.BURN_TOKENS,
-            directory.isTerminalOf(_projectId, IJBPaymentTerminal(_msgSender()))
+            JBPermissionIds.BURN_TOKENS,
+            directory.isTerminalOf(_projectId, IJBTerminal(_msgSender()))
         )
     {
         // There should be tokens to burn
         if (_tokenCount == 0) revert NO_BURNABLE_TOKENS();
 
         // Burn the tokens.
-        tokenStore.burnFrom(_holder, _projectId, _tokenCount);
+        tokens.burnFrom(_holder, _projectId, _tokenCount);
 
         emit BurnTokens(_holder, _projectId, _tokenCount, _memo, _msgSender());
     }
 
-    /// @notice Distributes all outstanding reserved tokens for a project.
+    /// @notice Sends a project's pending reserved tokens to its reserved token splits.
+    /// @dev If the project has no reserved token splits, or they don't add up to 100%, the leftover tokens are minted to the project's owner.
     /// @param _projectId The ID of the project to which the reserved tokens belong.
     /// @param _memo A memo to pass along to the emitted event.
-    /// @return The amount of minted reserved tokens.
-    function distributeReservedTokensOf(uint256 _projectId, string calldata _memo)
+    /// @return The amount of reserved tokens minted and sent.
+    function sendReservedTokensToSplitsOf(uint256 _projectId, string calldata _memo)
```

Comment:
Just wanted to point out how great this is- It gives so much more context.

Reply:
<3